bvt/op-machines-example.xml

```diff
@@ -15,6 +15,9 @@
       <ipmi-user>ADMIN</ipmi-user>
       <ipmi-password>admin</ipmi-password>
     </bmc>
+    <lspci>
```

Comment:
I'd prefer if we just had the filename in the lspci node rather than needing another node. Alternative could be to have filename specify it's in another file, or another node indicating it's inline.